server/openapi/api_resource_api_service.go

```diff
@@ -192,8 +192,8 @@ func (s *ResourceApiApiService) GetDataStore(ctx context.Context, dataStoreId st
 	// TODO - update GetDataStore with the required logic for this service method.
 	// Add api_resource_api_service.go to the .openapi-generator-ignore to avoid overwriting this service implementation when updating open api generation.
 
-	//TODO: Uncomment the next line to return response Response(200, DataStoreResource{}) or use other options such as http.Ok ...
-	//return Response(200, DataStoreResource{}), nil
+	//TODO: Uncomment the next line to return response Response(200, datastore{}) or use other options such as http.Ok ...
```

Comment:
should this be renamed? try running `make generate` to keep it in sync with the openapi files

Reply:
No 🤦 , I'm reverting this change.